### PR TITLE
Pg 5

### PR DESCRIPTION
**Pull request by aanchal-adk**

### CODE REVIEW

File: public/build/css/custom.css [hunk 1]
```diff
@@ -732,7 +732,11 @@ a:focus {
     background: transparent
 }
 .nav.child_menu {
+<<<<<<< HEAD
+    display: none;
+=======
     display: none
+>>>>>>> f8389074a368b5784249f97867a1548dae2e0081
```
Comment (pratishshr) [inline public/build/css/custom.css]: resolve this confict.

File: public/index.html [hunk 2]
```diff
@@ -10,9 +10,9 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico">
-     <link href="/vendors/bootstrap/dist/css/bootstrap.min.css" rel="stylesheet">
-     <link href="/vendors/font-awesome/css/font-awesome.min.css" rel="stylesheet">
-     <link href="/build/css/custom.min.css" rel="stylesheet">
+    <link href = "/vendors/bootstrap/dist/css/bootstrap.min.css" rel = "stylesheet">
+    <link href = "/vendors/font-awesome/css/font-awesome.min.css" rel = "stylesheet">
+    <link href = "/build/css/custom.min.css" rel = "stylesheet">
```
Comment (pratishshr) [inline public/index.html]: dont add extra spaces

File: src/app/App.js~ [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
```
Comment (pratishshr) [inline src/app/App.js~]: yo file chai k ho?

File: src/app/components/DatabaseList/databaseItem.js [hunk 4]
```diff
@@ -6,45 +6,67 @@ class DatabaseItem extends Component{
...
     };
   }
 
+  flagOn(){
+    this.setState({
+      plusIsNext:!this.state.plusIsNext
+    });
+  }
+  componentWillMount(){
+    console.log("TABLES");
```
Comment (pratishshr) [inline src/app/components/DatabaseList/databaseItem.js]: remove console.log

File: src/app/components/DatabaseList/databaseItem.js [hunk 5]
```diff
@@ -6,45 +6,67 @@ class DatabaseItem extends Component{
...
+    let data ={
+      query:"select * FROM information_schema.tables WHERE table_schema='public'",
+      dbname:this.props.dbname
+    };
+    httpUtil.post(`http://localhost:4553/api/queries`,data).then(response => {
+      console.log("items",response.data.reply.rows);
```
Comment (pratishshr) [inline src/app/components/DatabaseList/databaseItem.js]: remove console.log

File: src/app/components/DatabaseList/databaseItem.js [hunk 6]
```diff
@@ -6,45 +6,67 @@ class DatabaseItem extends Component{
...
     else{
-      console.log('yadayadayaaa');
-      let arr = [];
+      element.style.display = "none";
       this.setState({
-          tabList:[],
           plusIsNext:true
         })
-      console.log('adk',this.state.tabList);
+      console.log(this.state.tabList);
```
Comment (pratishshr) [inline src/app/components/DatabaseList/databaseItem.js]: .

File: src/app/components/DatabaseList/databaseItem.js [hunk 7]
```diff
@@ -6,45 +6,67 @@ class DatabaseItem extends Component{
...
     }
+    //this.flagOn();
```
Comment (pratishshr) [inline src/app/components/DatabaseList/databaseItem.js]: remove comment

File: src/app/components/DatabaseList/databaseList.js [hunk 8]
```diff
@@ -10,10 +10,14 @@ class DatabaseList extends Component{
...
   }
   componentWillMount(){
-    httpUtil.get(`http://localhost:4553/api?query=select * FROM pg_database where datistemplate=false and datname!='postgres'`,'postgres').then(response => {
-      // console.log('adk',response.data);
+    let data ={
+      query:"select * FROM pg_database where datistemplate=false and datname!='postgres'",
+      dbname:"postgres"
+    };
+    httpUtil.post(`http://localhost:4553/api/queries`,data).then(response => {
+      console.log(response.data.reply.rows);
```
Comment (pratishshr) [inline src/app/components/DatabaseList/databaseList.js]: .

File: src/app/App.js~ (hunk 3 above)
Comment (aanchal-adk) [reply src/app/App.js~]: yo chai tyo 'gedit' text editor bata kholda baneko thiyo hola ani delete garna birsechu. esbata delete garna milcha?

**Pull request merged**